app/src/main/java/com/example/oddhours/database/DatabaseHelper.kt

```diff
@@ -0,0 +1,97 @@
+package com.example.oddhours.database
+
+import android.content.ContentValues
+import android.content.Context
+import android.database.Cursor
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+
+class DatabaseHelper(context: Context):
+    SQLiteOpenHelper(context, dbName, null, 1){
+
+    val sqlCreateTable_job = "CREATE TABLE IF NOT EXISTS $jobTable (" +
+            "$job_ID_COL_1 INTEGER PRIMARY KEY, " +
+            "$job_Name_COL_2 TEXT, " +
+            "$job_Location_COL_3 TEXT);"
+    val sqlCreateTable_hours = "CREATE TABLE IF NOT EXISTS $hoursTable (" +
+            "$shift_ID_COL_1 INTEGER PRIMARY KEY, " +
+            "$shift_Date_COL_2 TEXT, " +
+            "$job_ID_COL_3 INTEGER, " +
+            "$start_Time_COL_4 TEXT, " +
+            "$end_Time_COL_5 TEXT, " +
+            "$hours_Worked_COL_6 INTEGER, " +
+            "FOREIGN KEY($job_ID_COL_3) REFERENCES $jobTable($job_ID_COL_1));"
+
+    val getJobs = "SELECT * FROM $jobTable"
+
+    //JobModel (JobId, JobName, JobLocation)
+    //ShiftsModel (ShiftID, ShiftDate, JobID, StartTime, EndTime, HoursWorked)
+
+    companion object {
+        val dbName = "oddHours"
+        val jobTable = "Job"
+        val job_ID_COL_1 = "JobID"
+        val job_Name_COL_2 = "JobName"
+        val job_Location_COL_3 = "JobLocation"
+        val hoursTable = "Hours"
```

Comment:
is it possible to change the hours table name to shifts? i figured if we're going to do it, it'll be easier to do it now rather than later. it's not a big deal though so i'm fine if we're not renaming it :)

Reply:
yes you're right that should be called shiftsTables, i'll make the change... missed that lol